test/e2e/run.sh

```diff
@@ -37,7 +38,11 @@ main() {
     run_e2e_tests "$@" | tee "$ARTEFACTS_DIR/e2e-tests-$CLUSTER_NAME.json"
   fi
   touch /tmp/done
-  sleep infinity
+  if [ "$(uname)" == "Linux" ] ; then
+    sleep infinity
+  else
+    sleep 86400
```

Comment:
Out of curiosity, why not sleep 24h on all platforms?

Reply:
+1 one for just one implementation that works cross platform. Naively I would think that 
```
while true; do
  sleep 1 
done
```
is the equivalent to `sleep infinity`?

Reply:
To give a bit of context, I wrote `sleep infinity` because it seemed like a clean and explicit way to wait forever but I missed that it wasn't cross-platform.

+1 for a more portable way, `while true; do sleep 42; done` or `tail -f /dev/null`.

Reply:
This has been updated, lmk your thoughts.